x-pack/plugins/viz_editor/public/frame/main/main.tsx

```diff
@@ -69,20 +72,47 @@ export function Main(props: MainProps) {
   const panelProps = {
     visModel: state.visModel,
     onChangeVisModel,
+    getSuggestionsForField,
```

Comment:
This is actually a little tricky and I wanted to talk to you about it- this is creating a 1:1 mapping between the Data Panel <-> field suggestions, instead of our ideal state where we have a 1:many mapping from Data Panel <-> [many field suggestions from all visualizations]

Reply:
We could build a multi-`getSuggestionsForField` which calls all the `getSuggestionsForField` of the individual editor plugins and flattens the result while prioritizing the ones of the currently active plugin